### DISCUSSION TITLE
Can't reference system environment variable

### DISCUSSION THREAD

**Discussion by strazan** [vercel/community#666] [Help] [answered]
Hey,

I need to create an env referencing a system environment variable, but I can't seem to find the option to select the type of env.

If I click edit at old system envs I get the option to choose type

<img width="760" alt="image" src="https://user-images.githubusercontent.com/43888637/174277628-bcfb0361-763b-4e70-ab11-4107fe6fde4b.png">

But when creating a new one, that option does not exist.

<img width="776" alt="image" src="https://user-images.githubusercontent.com/43888637/174278028-4908f441-9b30-4706-9f62-03e9e4f6fd7e.png">

**Comment by swarnava** [comment, marked as answer]
Hi,

Our team will reach out via ticket you have raised with us. Thanks!
